ui/raidboss/data/03-hw/raid/a11s.js

```diff
@@ -219,10 +225,10 @@
           return {
             en: 'Knockback Cleave; Face Outside',
             de: 'Rückstoß Cleave; nach Außen schauen',
+            fr: 'Poussée Cleave; Regardez à l\'extérieur',
```

Comment:
_Regardez vers l\'extérieur_ is better, don't you think ?

Reply:
J'ai honte, ça n'a aucun sens. Bon il était très tard.